HarfangLab/harfanglab/ingest/parser.yml

```
@@ -129,7 +129,26 @@ stages:
 
       - set:
           user.name: '{{json_event.message.impacted_users[0].user_name.replace("\\", " ")}}'
-        filter: "{{json_event.message.impacted_users | length > 0}}"
+        filter: "{{json_event.message.impacted_users | length == 1 }}"
```

In order to not break existing alert-filters or rules, I strongly suggest keeping the same filter as previously.
```suggestion
        filter: "{{json_event.message.impacted_users | length > 0 }}"
```